TruckMuncher/MapViewController.swift

```diff
@@ -64,6 +125,20 @@ class MapViewController: UIViewController, MKMapViewDelegate, CLLocationManagerD
         return annotationView
     }
     
+    func mapView(mapView: MKMapView!, didSelectAnnotationView view: MKAnnotationView!) {
+        var clusterAnnotation = view.annotation as? CCHMapClusterAnnotation
+        var truckLocationAnnotation = clusterAnnotation?.annotations.allObjects[0] as? TruckLocationAnnotation
+        
+        let tappedTruckIndex = truckLocationAnnotation!.index
```

Comment:
with the above two variables as optionals, forced unwrapping truckLocationAnnotation! here could cause an app crash in very rare corner cases.